contracts/YearnV2Strat.sol

```diff
@@ -0,0 +1,130 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity 0.7.3;
+
+import "./IStrat.sol";
+import "./IYToken.sol";
+import "./IVault.sol";
+import "./Timelock.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts/math/Math.sol";
+
+contract YTokenStrat is IStrat {
+    
+    using SafeMath for uint256;
+    using SafeERC20 for IERC20;
+    using SafeERC20 for IERC20Detailed;
+    IVault public vault;
+    IYToken public yToken;
+    IERC20Detailed public underlying;
+    Timelock public timelock;
+    uint public minWithdrawalCap; // prevents the owner from completely blocking withdrawals
```

Comment:
can set this to immutable for some gas saving

Reply:
ok